traversal/src/main/scala/overflowdb/traversal/Traversal.scala

```diff
@@ -47,6 +49,9 @@ class Traversal[A](elements: IterableOnce[A])
   def cast[B]: Traversal[B] =
     mapElements(_.asInstanceOf[B])
 
+  def collectAll[B <: A: ClassTag]: Traversal[B] =
```

Comment:
since you are adding docs, this is missing one ;)

Reply:
done, i've also added docs for more steps